website/pages/en/network/delegating.mdx

```diff
@@ -41,64 +55,92 @@ Another thing to consider is how to choose an Indexer wisely. If you choose an I
 
 ### Choosing a trustworthy Indexer with a fair reward payout for Delegators
 
-This is an important aspect to understand. First, let's discuss three very important values, which are the Delegation Parameters.
+In order to understand how to choose a trustworthy Indexer, you need to understand the Delegation Parameters.
+
+#### Delegation Parameters
 
-Indexing Reward Cut - The indexing reward cut is the portion of the rewards that the Indexer will keep for themselves. This means that if an Indexer's rewards are set to 100%, as a Delegator you will get 0 indexing rewards. If you see it set at 80% in the UI, that means as a Delegator, you will receive 20%. An important note - at the beginning of the network, Indexing Rewards will account for the majority of the rewards.
+- **Indexing Reward Cut** - The portion of the rewards the Indexer will keep for themselves.
+  - If an Indexer's rewards are set to 100%, as a Delegator, you will get 0 indexing rewards.
+  - If you see it set at 80% , as a Delegator, you will receive 20%.
+    > Note: At the beginning of the network, Indexing Rewards will account for the majority of the rewards.
 
 <div sx={{ mx: 'auto', maxWidth: '700px' }}>
   ![Indexing Reward Cut](/img/Indexing-Reward-Cut.png) *The top Indexer is giving Delegators 90% of the rewards. The
   middle one is giving Delegators 20%. The bottom one is giving Delegators ~83%.*
 </div>
 
-- Query Fee Cut - This works exactly like the Indexing Reward Cut. However, this applies explicitly to returns on the query fees the Indexer collects. It should be noted that at the start of the network, returns from query fees will be very small compared to the indexing reward. It is recommended that you pay attention to the network to determine when the query fees in the network will start to be more significant.
+- **Query Fee Cut** - This is the same as the Indexing Reward Cut, but it specifically applies to returns on the query fees the Indexer collects.
+
+  > Note: At the start of the network, returns from query fees will be very small compared to the indexing reward. It is recommended that you pay attention to the network to determine when the query fees will start to become more significant.
```

Comment:
I would also remove this note, or at least rephrase it to not mention "at the start of the network", since the network has been up for 3 years.